src/applications/vaos/tests/e2e/page-objects/DateTimeSelectPageObject.js

```diff
@@ -1,14 +1,18 @@
-import moment from 'moment';
+import { endOfMonth, format } from 'date-fns';
+import { formatInTimeZone, zonedTimeToUtc } from 'date-fns-tz';
 import { getTimezoneByFacilityId } from '../../../utils/timezone';
 import PageObject from './PageObject';
 
 export class DateTimeSelectPageObject extends PageObject {
   assertDateSelected(date, facilityId = 983) {
     const timezone = getTimezoneByFacilityId(facilityId);
-    const d = moment(date).tz(timezone);
+    const d = zonedTimeToUtc(date, timezone);
 
     cy.findByRole('radio', {
-      name: `${d.format('h:mm')} ${d.format('A')} option selected`,
+      name: `${formatInTimeZone(d, timezone, 'h:mm')} ${format(
```

Comment:
nit: this can be consolidated into a single format string.

Reply:
Done.